pkg/services/ngalert/state/template/funcs.go

```diff
@@ -89,3 +91,47 @@ func tableLinkFunc(data string) string {
 	expr := url.QueryEscape(q.Expr)
 	return fmt.Sprintf(`/explore?left={"datasource":%[1]q,"queries":[{"datasource":%[1]q,"expr":%q,"instant":true,"range":false,"refId":"A"}],"range":{"from":"now-1h","to":"now"}}`, datasource, expr)
 }
+
+func contains[T comparable](slice []T, elem T) bool {
+	for _, e := range slice {
+		if e == elem {
+			return true
+		}
+	}
+	return false
```

Comment:
I think this can be replaced by `slices.Contains`

Reply:
It can. I didn't think to check if we were using it elsewhere and was hesitant to bring in an experimental module.